CONTRIBUTING.md

```diff
@@ -157,11 +157,14 @@ These are the steps to create a new release:
 
 ##  Cross-Arch Testing
 
-This project has built-in support for testing across multiple architectures. Currently, our CI conducts tests on `arm64` machines using QEMU emulation. We also have the flexibility to expand this support to include additional architectures in the future.
-
-This project containers dockerfiles that can be used in the CI to test the
-python package in non-native machine architectures, e.g., `arm64`.
-The dockerfiles exist in the directory `.github/containers/nox-cross-arch`, and
+This project has built-in support for testing across multiple architectures.
+Currently, our CI conducts tests on `arm64` machines using QEMU emulation. We
+also have the flexibility to expand this support to include additional
+architectures in the future.
+
+This project containers Dockerfiles that can be used in the CI to test the
```

Comment:
```suggestion
This project contains Dockerfiles that can be used in the CI to test the
```

Reply:
looks like this joke about containering the docker files is still there in 7.2.  I'm inclined to letting it stay, tbh. :D

Reply:
@llucax  should this typo have been fixed? It's also fine with me to keep it as it is

Reply:
Damn, I didn't regenerate the templates, just did manual updates, I must have forgotten about this one. Will fix.

Reply:
It should be hopefully fixed now.